### PR TITLE
feat(common): [components] add WTextButton

### PR DESCRIPTION
**Pull request by disha1202**
part of #272 

Screenshot from 2023-01-21 13-44-54

### CODE REVIEW

File: src/common/components/WTextButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  content: string;
+  /** Callback function for click event. */
+  handlerClick: CallableFunction;
+}): JSX.Element {
+  // ---------------------------------------------------------------------------
+
+  return (
+    <div className="w-button" onClick={() => props.handlerClick()}>
+      {props.content}
+    </div>
```
Comment (meganindya) [inline src/common/components/WTextButton/index.tsx]: use `button` element

File: src/common/components/WTextButton/index.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+.w-button {
+  height: 32px;
+  width: 75px;
+  color: $mb-accent;
+  background-color: $background-light;
+  border-radius: $s-border-radius;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 14.4px;
```
Comment (meganindya) [inline src/common/components/WTextButton/index.scss]: use `rem` units for font-size (`0.9rem` here)

File: src/common/components/WTextButton/index.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+@import '@/common/scss/colors.scss';
+@import '@/common/scss/sizes.scss';
+
+.w-button {
+  height: 32px;
+  width: 75px;
```
Comment (meganindya) [inline src/common/components/WTextButton/index.scss]: width must be `auto` (can remove attribute) otherwise it won't adapt to text

**Review by meganindya — CHANGES_REQUESTED** [review]

**Comment by meganindya** [comment]
You should not merge default branch into a feature branch. You should always rebase.

However, don't rebase for this PR (I'm making some changes to the dependencies so it'll conflict)

**Comment by disha1202** [comment]
@meganindya Made the requested changes.
Please take a look.

**Review by meganindya — APPROVED** [review]

**Pull request merged**